WordPress/src/main/res/layout/new_site_creation_verticals_custom_suggestion_item.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                                             xmlns:app="http://schemas.android.com/apk/res-auto"
+                                             xmlns:tools="http://schemas.android.com/tools"
+                                             android:id="@+id/container"
+                                             android:layout_width="match_parent"
+                                             android:layout_height="wrap_content"
+                                             android:background="@color/white"
+                                             android:foreground="?selectableItemBackground"
+                                             android:minHeight="@dimen/two_line_list_item_height">
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/margin_extra_large"
+        android:layout_marginStart="@dimen/margin_extra_large"
+        android:layout_marginTop="@dimen/margin_extra_large"
```

Comment:
Nitpick: Instead of having a top margin here and the bottom margin in the subtitle, how about we have the margins in the layout itself, so it's clearer how the layout looks at first glance?

P.S: I might be the only one that's actually reading xml instead of looking at the design to understand what's going on, so feel free to ignore this.

Reply:
I also prefer adding paddings to the parent rather than specifying margins in the children. However, afaik we can't do it unless we get rid of the divider:D.